osquery/tables/system/windows/logon_sessions.cpp

```diff
@@ -0,0 +1,142 @@
+// clang-format off
+#include <Windows.h>
+#include <NTSecAPI.h>
+#include <sddl.h>
+#include <tchar.h>
+#include <vector>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/process_ops.h"
+// clang-format on
+
+#define CHECK_BIT(var, pos) ((var) & (1 << (pos)))
+#define JAN_1_1601_TO_JAN_1_1970 116444736000000000
+#define HUNDREDNANOSECONDS_TO_SECONDS 10000000
+
+namespace osquery {
+namespace tables {
+LONGLONG filetimeToUnixtime(const LARGE_INTEGER& ft) {
+  LARGE_INTEGER date, adjust;
+  date.HighPart = ft.HighPart;
+  date.LowPart = ft.LowPart;
+  adjust.QuadPart = JAN_1_1601_TO_JAN_1_1970;
+  date.QuadPart -= adjust.QuadPart;
+  return date.QuadPart / HUNDREDNANOSECONDS_TO_SECONDS;
```

Comment:
This function is copied from https://github.com/facebook/osquery/blob/dad25b89a9a898690ca9a3c3a5caf4ee76e7c5c0/osquery/filesystem/windows/fileops.cpp#L1463, but I have to convert a LARGE_INTEGER instance instead of a FILETIME instance.

Reply:
Ah I see. In that case @jaredcatkinson I'd recommend extrending the `fileops.cpp` to contain this implementation and make use of that function by declaring it here in the `osquery` namespace.

Reply:
FYI I needed a similar function. It's part of this pending PR https://github.com/facebook/osquery/pull/4613/files#diff-b95aff49cefcd24ad84cd578816de3f0R1476

Reply:
This PR was merged so you should be able to use https://github.com/facebook/osquery/blob/master/osquery/filesystem/windows/fileops.cpp#L1477 now